src/System.IO.FileSystem/src/System/IO/FromApp.Interop.cs

```diff
@@ -0,0 +1,150 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Noted. Note that I do plan to move things there for RS4- we'll have the "permanent" APIs then.